### ISSUE TITLE
Won't work when a middleware function is specified

### ISSUE THREAD

**Issue by stephenwil** [vitkon/form-container#27]
## Expected Behavior
```
const config = {
  initialModel: {},
  middleware: (props: any) => { console.log(props)},
  onInputBlur: () => {}
}
const validators = concat([], requiredValidators);
export default connectForm(validators, config)(CreateClient);
```
*No Errors in console*
## Actual Behavior
Errors in console:
```
TypeError: Cannot read property 'inputs' of undefined
inferRulesFromAttributes
node_modules/form-container/dist/index.js:17496
  17493 |     return __assign({}, errors, field);
  17494 | }, {}); };
  17495 | var inferRulesFromAttributes = function (rules, _a) {
> 17496 |     var inputs = _a.inputs;
  17497 |     var extendedRules = rules.slice();
  17498 |     lodash_1.forEach(inputs, function (input) {
  17499 |         if (lodash_1.get(input, 'validity.valid') === false) {
```

## Steps to Reproduce the Problem

  See above

## Specifications

  - Version: 0.1.0
  - Platform: Mac OS Sierra
  - Subsystem: Node v.8.9.1

**Comment by mattdell** [comment]
I can reproduce this. Will investigate.

**Comment by mattdell** [comment]
Pull request raised in #30.

TL;DR middleware needs to return the model
